ls1intum/Apollon_standalone
# `Development`: Update Apollon version and fix exports

### Motivation and Context
This PR updates Apollon version and fixes export failures.

### Steps for Testing
1) Go to [test server](https://test1.apollon.ase.in.tum.de/)
2) Check that export functionality(svg, png, pdf) is working as expected.
